content/blog/tinacms-ui-whats-next.md

```diff
@@ -0,0 +1,53 @@
+
+Now that Open Authoring with [tinacms.org](http://tinacms.org/) is published. We're getting ready to make it available for everyone. Kendall has already moved over the [new Inline Editing components](https://github.com/tinacms/tinacms/pull/871) and figuring out how the Toolbar will fit in the TinaCMS ecosystem is next. This post lays out the plan for how that will be done.
+
+## `tinacms` as a quick-start package
+
+Currently, the `tinacms` package implements a number of features directly. The Sidebar, Modals, Alerts, and Screen Plugins are all defined directly here. After this change, `tinacms` will simply be the "quick-start" package for developers looking to get started with Tina as fast as possible. The sole purpose of the `tinacms` package will be to provide a pre-configured CMS instance along with a few simple component that adds the CMS, Alerts, Modals, Screens, and either the Sidebar or the Toolbar to the website. This change will give new-comers to Tina the ability to get started quickly, but it will also give other developers the ability to optimize and modify Tina to fit their needs.
+
+A large problem with `tinacms` is the impact it has on the bundle size of the developer's website. This will be made worse when we start adding more features to the toolbar. By breaking these pieces up, developers can solve this problem by taking advantage of dynamic imports and code splitting in their application.
+
+This separation will give developers the ability to use opt-out of the pre-defined UI and roll their own. This will give developers the option to deeply integrate the CMS into their website and opens Tina up to the possibility of great UI/UX innovations.
+
+## Upcoming Packages
+
+The following packages will be introduced in this process:
+
+* `@tinacms/media`: the API for interacting with the [CMS media store](https://tinacms.org/docs/media)
+* `@tinacms/alerts`: the API for creating [CMS alerts](https://tinacms.org/docs/cms/alerts)
+* `@tinacms/react-alerts`: the components for rendering CMS _alerts_
+* `@tinacms/react-forms`: the components used to automatically build _forms_
+* `@tinacms/react-modals`: the components for creating _modals_ for the CMS
+* `@tinacms/react-sidebar`: the sidebar components
+* `@tinacms/react-toolbar`: the toolbar components
+* `@tinacms/react-fields`: Primitive field plugins i.e. Text, Textarea, Select, Number, Toggle, Block, Group, Group List
+* `react-tinacms-color`: Color Field plugin
+* `react-tinacms-date`: Date Field plugin
+* `react-tinacms-editor`: HTML and Markdown field plugins + the Wysiwyg itself
+* `react-tinacms-image`: Image plugin
+
+### Why separate `alerts` and `react-alerts`?
+
+TinaCMS is designed to be framework agnostic. We started with React as the main implementation because it is the framework that the team is most familiar with. This separation allows us to separate the domain from the view library so that TinaCMS will be accessible to more use-cases.
```

Comment:
```suggestion
TinaCMS is designed to be framework agnostic. We started with React as the main implementation because it is the framework that the team is most familiar with. This separation allows us to separate the domain from the view library so that TinaCMS will be accessible to more use cases.
```